frontend-service/src/main/java/com/amazonaws/blox/frontend/models/Environment.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "LICENSE" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.amazonaws.blox.frontend.models;
+
+public final class Environment {
+  private final String name;
+
+  public final String getName() {
```

Comment:
Same as above, we didn't have lombok in the project when I started writing this, will update.